python/src/main/parser.py

```diff
@@ -2,14 +2,11 @@
 from python.src.main.Item import Item
 
 
-class Parser(object):
-    def __init__(self, row):
-        self.row = row
+def create_person(row):
```

Comment:
Parser doesn't need to be an object and it's methods are 'static' meaning they don't hold any state.